crates/fullnode/src/runner.rs

```diff
@@ -517,89 +496,7 @@ where
-                                    );
-                                }
-                            } else {
-                                warn!("Force transactions are not implemented yet");
-                                // TODO: This is where force transactions will land - try to parse DA data force transaction
-                            }
-                        });
-
-                    pending_zk_proofs.extend(zk_proofs);
-                    pending_sequencer_commitments.extend(sequencer_commitments);
-
-                    for (index, zk_proof) in pending_zk_proofs.clone().iter().enumerate() {
-                        match self.process_zk_proof(l1_block.clone(), zk_proof.clone()).await {
-                            Ok(()) => {
-                                pending_zk_proofs.remove(index);
-                            },
-                            Err(e) => {
-                                match e {
-                                    SyncError::MissingL2(msg, start_l2_height, end_l2_height) => {
-                                        warn!("Could not completely process ZK proofs. Missing L2 blocks {:?} - {:?}. msg = {}", start_l2_height, end_l2_height, msg);
-                                    },
-                                    SyncError::Error(e) => {
-                                        error!("Could not process ZK proofs: {}...skipping", e);
-                                        pending_zk_proofs.remove(index);
-                                    }
-                                }
-                            }
-                        }
-                    }
-
-                    for (index, sequencer_commitment) in pending_sequencer_commitments.clone().iter().enumerate() {
-                        match self.process_sequencer_commitment(l1_block.clone(), sequencer_commitment.clone()).await {
-                            Ok(()) => {
-                                pending_sequencer_commitments.remove(index);
-                            },
-                            Err(e) => {
-                                match e {
-                                    SyncError::MissingL2(msg, start_l2_height, end_l2_height) => {
-                                        warn!("Could not completely process sequencer commitments. Missing L2 blocks {:?} - {:?}, msg = {}", start_l2_height, end_l2_height, msg);
-                                    },
-                                    SyncError::Error(e) => {
-                                        error!("Could not process sequencer commitments: {}... skipping", e);
-                                        pending_sequencer_commitments.remove(index);
-                                    }
-                                }
-                            }
-                        }
-                    }
+                   self.run_inner(&mut pending_sequencer_commitments,&mut pending_zk_proofs, l1_block).await;
```

Comment:
```suggestion
                   self.process_l1_block(&mut pending_sequencer_commitments,&mut pending_zk_proofs, l1_block).await;
```
run in this case indicates this is a worker which this is not.

The L2 block below calls `process_l2_block`